Header header
string source
string destination
aut_exploration/Agent_toMaster data
